tests/integration_test.rs

```diff
@@ -1,6 +1,53 @@
 use egglog::{extract::DefaultCost, *};
 use egglog_ast::span::{RustSpan, Span};
 
+#[test]
+fn globals_missing_prefix_warns_by_default() {
+    let _ = env_logger::builder().is_test(true).try_init();
+
+    let mut egraph = EGraph::default();
+    egraph
```

Comment:
Could this test that the warning is triggered somehow?

Reply:
It prints to stderr so I'm not sure how to
Right now it just tests no error happens

Reply:
Ah yeah, maybe if you use warm then you can verify that it warns in the test?